### PR TITLE
Update Spanish.properties

### PR DESCRIPTION
**Pull request by ArchDuque-Pancake**
Have ye ever heard the tragedy of darth Plagueis the Wise?

### CODE REVIEW

**Comment by xlenstra** [comment]
Placeholder `[stat]` not found in `Spanish` for entry `Earn [amount]% of [unitType] unit's [param] as [stat] when killed within 4 tiles of a city following this religion`

File: android/assets/jsons/translations/Spanish.properties [hunk 1]
```diff
@@ -564,8 +560,7 @@ Pillage = Saquear
 Are you sure you want to pillage this [improvement]? = ¿Estás seguro de que quieres Saquear este [improvement]?
 Create [improvement] = Crear [improvement]
 Start Golden Age = Empezar edad dorada
- # Requires translation!
-Show more = 
+Show more = Motrar más
```
Comment (SomeTroglodyte) [inline android/assets/jsons/translations/Spanish.properties:563]: mostrar?

File: android/assets/jsons/translations/Spanish.properties [hunk 2]
```diff
@@ -238,14 +238,10 @@ Cultural = Cultural
...
- # Requires translation!
-Height = 
- # Requires translation!
-Width = 
- # Requires translation!
-Radius = 
- # Requires translation!
-Enable Religion = 
+Height = Altura
+Width = Ancho
```
Comment (SomeTroglodyte) [inline android/assets/jsons/translations/Spanish.properties:242]: anchura? width of the screen/window is what's meant here

File: android/assets/jsons/translations/Spanish.properties [hunk 3]
```diff
@@ -3345,20 +3234,15 @@ Send your best explorers on a quest to discover Natural Wonders. Nobody knows th
 
 #################### Lines from Religions from Civ V - Vanilla ####################
 
- # Requires translation!
-Buddhism = 
+Buddhism = Budísmo
```
Comment (SomeTroglodyte) [inline android/assets/jsons/translations/Spanish.properties:3237]: Verdad? 'í' y no 'i'?

File: android/assets/jsons/translations/Spanish.properties [hunk 4]
```diff
@@ -3582,16 +3466,14 @@ Desert = Desierto
...
+Occurs in chains at high elevations = Ocurre en cadenas a gran altura
 Mountain = Montañas
 
 Snow = Nieve
 
 [amount] Strength for cities built on this terrain = [amount] de Fuerza para ciudades asentadas en este terreno
 [amount] Sight for [param] units = [amount] de Vista para unidades [param]
- # Requires translation!
-Occurs in groups around high elevations = 
+Occurs in groups around high elevations = Ocurre en grupos a grandes alturas
```
Comment (SomeTroglodyte) [inline android/assets/jsons/translations/Spanish.properties:3476]: Dificil. Se trata de las colinas, y dice que se agrupan alrededor de las montan(~)as... Por eso "around". Pero no es muy importante. Lo ves en la Civilopedia solamente - creo.

File: android/assets/jsons/translations/Spanish.properties (hunk 1 above)
Comment (ArchDuque-Pancake) [reply android/assets/jsons/translations/Spanish.properties:563]: Ok, I corrected it in the new one

File: android/assets/jsons/translations/Spanish.properties (hunk 2 above)
Comment (ArchDuque-Pancake) [reply android/assets/jsons/translations/Spanish.properties:242]: I think it works like that

**Review by SomeTroglodyte — APPROVED** [review]
Buen trabajo...

**Pull request closed without merging**